### ISSUE TITLE
Please add LICENSE file

### ISSUE THREAD

**Issue by eleweek** [rcaloras/bash-preexec#3]
Hi, there is no LICENSE file, which means that:

>  you retain all rights to your source code and that nobody else may reproduce, distribute, or create derivative works from your work. This might not be what you intend

https://help.github.com/articles/open-source-licensing/

Could you please add a LICENSE file? If you don't really care about how people use the code, choose something permissive like e.g. MIT license

**Comment by rcaloras** [comment]
I'm interested to hear how people are using it, but definitely want it to be permissive. Enjoy!

**Comment by eleweek** [comment]
Thank you very much! 

> but definitely want it to be permissive. 

That's what I figured! You have a really nice documentation.

> I'm interested to hear how people are using it

Right now I'm implementing this: https://github.com/eleweek/histsync (very alpha stage) -- "Cloud Bash History" app. I needed a way to upload a command when it is executed. 

So I tried $PROMPT_COMMAND, but I ran into multiple issues(e.g. commands get uploaded when you create a new shell). It is not the same thing as preexec(). Preexec() actually is a lot better solution!
